### PR TITLE
babel correctly bundles dep names ending with !

### PR DESCRIPTION
**Pull request by npbenjohnson**
#555

### CODE REVIEW

File: test/test-build.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ var builder = new Builder('test/fixtures/test-tree', 'test/fixtures/test-tree.co
 
 function testPhantom(html) {
   return new Promise(function(resolve, reject) {
-    spawn('node_modules/.bin/mocha-phantomjs' + (process.platform.match(/^win/) ? '.cmd' : ''), [html], { stdio: 'inherit' })
+    spawn(path.resolve('node_modules/.bin/mocha-phantomjs' + (process.platform.match(/^win/) ? '.cmd' : '')), [html], { stdio: 'inherit' })
```
Comment (guybedford) [inline test/test-build.js]: Did this fix the tests in Windows. @arackaf perhaps this was the fix we were looking for all along!?

Comment (npbenjohnson) [inline test/test-build.js]: It fixed 10 of them I think, so if that was your problem then yeah. I have no idea why it does, was a complete shot in the dark.

**Comment by arackaf** [comment]
@guybedford  I really can't imagine what I did to my setup to have such problems - it seems as though right at commit 39f07d2c792c11e4e81677b29058c8285c8b1772 from you on 21 April, everything goes to hell for me - the original 6 tests keep failing, and tons more.  Here are some screenshots 

image

image

and much more of the same.

**Comment by arackaf** [comment]
Running the tests from master appear to have some different problems

image

And yes, I was definitely running npm install from each new head I checked out.

**Comment by guybedford** [comment]
@arackaf unfortunately the above are all bad timing on breaking commits I added to master, these should be fixed now if you're able to test out the original path.resolve thing. Thanks for taking the time to investigate!

**Comment by arackaf** [comment]
Very nice - I merged this PR, and all the tests pass!  Sweet. 

btw - I'm still seeing massive failures off of your master - and I see nothing new after fetching - you said this was fixed, so just fyi.

**Comment by guybedford** [comment]
@arackaf excellent thanks so much for confirming this! I've merged in https://github.com/systemjs/builder/commit/789a66e138e11b92236f5a1440d66d8f21a1cf78. 

None of the other changes on master have to do with pathing (aka things that break in windows), and all the errors you posted recently were ones that I had seen myself and fixed. If there's something else happening here do let me know.

**Comment by arackaf** [comment]
> were ones that I had seen myself and fixed

Are those fixes checked into master?  If so, that's what I was running.  If not, what's a good place to merge from an test again?

**Comment by arackaf** [comment]
Scratch that - latest on master passes all the tests.  Thanks much!

**Comment by guybedford** [comment]
👍

**Pull request closed without merging**